unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ParcelScene.cs

```diff
@@ -405,7 +405,7 @@ public void SetEntityParent(string entityId, string parentId)
 
             if ( DCLCharacterController.i != null )
             {
-                if (parentId == "FirstPersonCameraEntityReference" || parentId == "PlayerEntityReference") // PlayerEntityReference is for compatibility purposes
+                if (parentId == "FirstPersonCameraEntityReference".GetHashCode() || parentId == "PlayerEntityReference".GetHashCode()) // PlayerEntityReference is for compatibility purposes
```

Comment:
Can this magic hash codes be constants? I'd like to have them outside of the range of "user entities", maybe a negative number or super high number. I consider having GetHashCode is a dangerous blackbox

Reply:
Yes, I can move them to constant, however we should not use a negative or super high number rather than GetHashCode, because we can end using another string by accident. GetHashCode is the best way to ensure retrocompatibility right now.

Reply:
@menduz In the "legacy" scene runtime, the 10 first id numbers are reserved for these cases:
https://github.com/decentraland/kernel/blob/main/packages/scene-system/sdk/SceneRuntime.ts#L62

Reply:
Please let's create a "conversion function" that takes the strings and gives us predictable constant values for well known entities such as "0" and the others mentioned across the PR. That function should be used to convert string->long without risking any protocol due to magic hashcodes

Reply:
I'll take the chance to write down a spec document for the SDK entities today to have this thing documented as part of the "Decentraland Protocol"

Reply:
I already wrote it down on another comment but just in case: the conversion is a base conversion, ids are just constructed by converting the incremental ids to base 36. In the actual implementation 0 to 10 are reserved for special cases. I recognize that only reserving 10 ids can be a limitation so you may want to change it for the SDK 7, just stating how it works right now.

Reply:
As you wish, I think that get hash has more performance than convert to base 10, but just let me know which one you prefer and I will change it